test/cmd/legacy-script.sh

```diff
@@ -29,6 +29,7 @@ KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
 # source "${KUBE_ROOT}/hack/lib/test.sh"
 source "${KUBE_ROOT}/test/cmd/apply.sh"
 source "${KUBE_ROOT}/test/cmd/apps.sh"
+source "${KUBE_ROOT}/test/cmd/auth.sh"
```

Comment:
```suggestion
source "${KUBE_ROOT}/test/cmd/authentication.sh"
```

Reply:
You need to call `run_exec_credentials_tests` in this file to actually run the tests (see other examples later in this file).

The [CI job](https://prow.k8s.io/pr-history/?org=kubernetes&repo=kubernetes&pr=91745) `pull-kubernetes-integration` should show the test running in the [raw build log](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/91745/pull-kubernetes-integration/1273332842755002372).

Reply:
Oh, right, good catch!